media-libs/taglib/taglib2.0-2.0.recipe

```diff
@@ -77,6 +77,12 @@ INSTALL()
 	fixDevelopLibDirReferences $binDir/taglib-config
 	sed -i -e "s,\/.*/include,$includeDir," $binDir/taglib-config
 
+	# Fix usage of hardcoded "/include":
+	sed -e "s|\(\${_IMPORT_PREFIX}\)/include|\1/$relativeIncludeDir|" -i \
+		$libDir/cmake/taglib/taglib-targets.cmake
+	sed -e "s|\(\${_IMPORT_PREFIX}\)/include/taglib|\1/$relativeIncludeDir/taglib|" -i \
+		$libDir/cmake/taglib/taglib-targets.cmake
```

Comment:
```suggestion
	sed -e "s|\(\${_IMPORT_PREFIX}\)/include|\1/$relativeIncludeDir|" \
		-e "s|\(\${_IMPORT_PREFIX}\)/include/taglib|\1/$relativeIncludeDir/taglib|" \
		-i $libDir/cmake/taglib/taglib-targets.cmake
```

You can combine multiple sed expressions in one command if you're changing the same file.  Not a big deal and probably not worth changing.

Reply:
Thanks! +1